CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-04-23
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
```

Comment:
[AAD workload identity](https://learn.microsoft.com/azure/aks/workload-identity-overview) is now Generally Available. should we not keep it as it technically just released and maybe good information for awareness??